src/shared/components/mutationMapper/MutationMapper.tsx

```diff
@@ -171,6 +186,10 @@ export default class MutationMapper<P extends IMutationMapperProps> extends Defa
                 trackDataStatus={this.trackDataStatus}
                 onTrackVisibilityChange={this.onTrackVisibilityChange}
                 getLollipopColor={getColorForProteinImpactType}
+                filterResetPanel={
+                    !this.props.store.dataStore.showingAllData && this.filterResetPanel ?
```

Comment:
could you just pass filterResetPanel whether it's defined or not?  just kind confusing

Reply:
The problem here is, for some reason, we initially defined `filterResetPanel` as `JSX.Element | null` instead of `JSX.Element | undefined`. When I just pass it, TS complains that `null` cannot be assigned to `undefined`. 

For now just updated the check to `this.filterResetPanel !== null`, but we should probably revisit those function definitions (both here and in `react-mutation-mapper`)